.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
-.code-workspace
\ No newline at end of file
+.code-workspace
+content-engine/.DS_Store
```

Comment:
You should ignore .DS_Store everywhere, not just in a subfolder.

Reply:
Also ignore `.idea` and [resync the repo with the .gitignore](https://stackoverflow.com/questions/7075923/resync-git-repo-with-new-gitignore-file)